docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -1072,9 +1072,10 @@ The downside of this approach is that CDI Portable Extensions cannot be supporte
 Nevertheless, most of the functionality can be achieved using Quarkus xref:writing-extensions.adoc[extensions].
 See the xref:cdi-integration.adoc[integration guide] for more information.
 
-== Development Mode
+[[dev_mode]]
+== Dev mode
```

Comment:
This kind of change breaks existing links, please let's not do that :(

```suggestion
[[dev_mode]]
== [[development-mode]] Dev mode
```

Reply:
We could do that but I'm pretty sure nobody will do it when adding an explicit anchor in the future so...

Reply:
I'm pretty sure "others don't do it" has never stopped you from doing things right :)